pyproject.toml

```diff
@@ -1,20 +1,87 @@
+[project]
+name = "nipy"
+dynamic = ['version']
+license = {file = "LICENSE"}
+description  = 'A python package for analysis of neuroimaging data'
+readme = 'README.rst'
+classifiers = ["Development Status :: 3 - Alpha",
+               "Environment :: Console",
+               "Intended Audience :: Science/Research",
+               "License :: OSI Approved :: BSD License",
+               "Operating System :: OS Independent",
+               "Programming Language :: Python",
+               "Topic :: Scientific/Engineering"]
+dependencies = [
+    'numpy>=1.22',
+    'scipy>=1.8',
+    'nibabel>=3.2',
+    'sympy>=1.9',
+    'transforms3d'
+]
+
+[[project.maintainers]]
+name = 'nipy developers'
+email = 'neuroimaging@python.org'
+
+[project.urls]
+homepage = 'https://nipy.org/nipy'
+documentation = 'http://nipy.org/nipy/documentation.html'
+source = 'https://github.com/nipy/nipy'
+download = 'https://pypi.org/project/nipy/#files'
+tracker = 'https://github.com/nipy/nipy/issues'
+
+[project.optional-dependencies]
+build = [
+    'meson-python>=0.13',
+    'wheel',
+    'setuptools>=67',
+    'ninja',
+    'Cython>=3',
+    'numpy>=1.22',
+    'spin==0.3',
+    'build',
+]
+default = [
+    'numpy>=1.22',
+    'scipy>=1.8',
+    'nibabel>=3.2',
+    'sympy>=1.9',
+    'transforms3d'
+]
```

Comment:
IMO build and default don't make much sense. `nipy[default]` is equivalent to `nipy` and now you have duplication.

I'm not sure what `nipy[build]` is adding, since to install it you will already need something capable of building, while also duplicating (and extending?) the `build-system.requires` table.

Reply:
@stefanv - any thoughts?  I copied this brainlessly from scikit-image: https://github.com/scikit-image/scikit-image/blob/main/pyproject.toml#L46

Reply:
@stefanv - any reason not to just delete the `build` and `default` sections of the `optional-dependencies`?